kieserver/image.yaml

```diff
@@ -200,6 +200,12 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_JBPM_CLUSTER"
+    example: "true"
+    description: "Enable the KIE Server Jbpm clustering for processes fail-over. If the number of replicas is less than two, one additional replica will be started."
+  - name: "KIE_SERVER_JBPM_CLUSTER_TRANSPORT_LOCK_TIMEOUT"
+    value: "60000"
+    description: "Jbpm Cache transport lock timeout, value in milliseconds. The default value is 6000 ms."
```

Comment:
same comments as in https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/414

Reply:
changed

Reply:
the default value on the description seems to be not correct, please fix it and for example, try to use a different value than default.